dist/Tie-File/t/26_twrite.t

```diff
@@ -306,54 +310,7 @@ sub try {
 }
 
 
-
```

Comment:
I couldn't find any evidence this subroutine was used. I did this removal in multiple tests.